src/commitment_scheme.rs

```diff
@@ -1,45 +1,30 @@
-use std::borrow::Cow;
+use std::{borrow::Cow, ops::Mul};
 
-use algebra::{bls12_381::{Fr, g1::G1Projective}, Bls12_381};
-use algebra_core::curves::AffineCurve;
-use ff_fft::polynomial::{DensePolynomial as Polynomial};
```

Comment:
this is replaced by: use ark_poly::{univariate::DensePolynomial as Polynomial, DenseUVPolynomial};